ingq/book/infra/pagination/cursor.py

```
@@ -0,0 +1,111 @@
+        # TODO: 추후 구현 예정
+        elif order_strategy in [
+            OrderStrategy.BOOKMARK_COUNT_DESC,
+            OrderStrategy.BOOKMARK_COUNT_ASC,
+        ]:
+            return None
+
+        elif order_strategy in [
+            OrderStrategy.VIEW_COUNT_DESC,
+            OrderStrategy.VIEW_COUNT_ASC,
+        ]:
+            return None
```

Ruff (0.8.2)

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)